### ISSUE TITLE
option to turn off copied to clipboard notificacions

### ISSUE THREAD

**Issue by ezjones** [termux/termux-app#149]
Is there a way to turn off notifications when something gets copied to the clipboard? Right now every time I press META-backspace on emacs, which happens a lot, I see a notification popup.

Other than that this is amazing.

**Comment by lynnard** [comment]
Just want to chime in to say that this would be very useful for users who do a lot of text editing in editors. I'm using spacemacs and every time I yank a chunk of text a wall of notification appears obstructing almost my entire view.

In short, please add this in!

**Comment by fornwall** [comment]
The updated version `0.49` of Termux, which has just been released and should be available as an update shortly, has removed the toast notification on clipboard text completely (we can bring it back as an option if there is enough demand for it).

Thanks for reporting!

**Comment by Mihara** [comment]
Did something else change with this? Looking at the commit it doesn't *look* like it, but it behaves somewhat differently now: If I copy something into the clipboard, Emacs does not appear to update its kill ring. I.e. using keyboard commands to paste pastes whatever Emacs had in its kill ring instead before I copied it from wherever else. If I use long press -> paste, it gets pasted *and* the contents of the kill ring is updated, i.e. Emacs yank commands start pasting the new paste.

I'm pretty sure it worked before, though whether it's a change in Termux or the fact that my tablet got updated to Nougat, I cannot quite say.
